examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -394,12 +391,16 @@ def run_benchmarks(args: argparse.ArgumentParser):
 
             # Enforce DATASETS_DIR set only if this benchmark needs real data
             if (not "gen" in benchmark_name) and (not "synth" in benchmark_name):
```

Comment:
```suggestion
            if ("gen" not in benchmark_name) and ("synth" not in benchmark_name):
```

Reply:
😂 Agreed, reads better this way. I'm was thinking in the `!(bool)` as opposed to `bool !=` way.